### PR TITLE
Guide: DCA Adjustments

### PR DESCRIPTION
**Pull request by fkaminski**
Starting Samples (DE).
(Not yet categorized. Will be easier when more content is available)

### CODE REVIEW

**Comment by fkaminski** [comment]
As discussed, additional categorization would be useful. Suggestion: 
Categorization by table name e.g. tl_content.php, tl_news.php etc..?

**Comment by fiedsch** [comment]
> Categorization by table name e.g. tl_content.php, tl_news.php etc..?

I'm not sure if this is a good idea, as many adjustments are the same for all tables. Think of "making a field mandatory".
So maybe rather categorization by "type of change" (without having an idea how this could be called).

**Comment by fkaminski** [comment]
> ... without having an idea how this could be called)

Me too :-)

File: docs/manual/guides/dca-adjustments.de.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+tags: 
+    - "DCA"
+---
+
+
+Das Contao »[Data Container Array](https://docs.contao.org/dev/reference/dca/)« (DCA) bietet zahlreiche, 
+praktische Konfigurationsmöglichkeiten. Hier findest du eine Auswahl hilfreicher Beispiele.
+
+Ab Contao 4.9 werden die jeweiligen Anpassungen im Verzeichnis »contao/dca« erwartet. Falls die Verzeichnisse noch 
+nicht existieren, musst du diese zunächst erstellen. Für jede Contao Tabelle benötigst du eine eigene Datei,
```
Comment (netzarbeiter) [inline docs/manual/guides/dca-adjustments.de.md]:
```suggestion
nicht existieren, musst du diese zunächst erstellen. Für jede Contao-Tabelle benötigst du eine eigene Datei, 
```

File: docs/manual/guides/dca-adjustments.de.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+
+Das Contao »[Data Container Array](https://docs.contao.org/dev/reference/dca/)« (DCA) bietet zahlreiche, 
+praktische Konfigurationsmöglichkeiten. Hier findest du eine Auswahl hilfreicher Beispiele.
+
+Ab Contao 4.9 werden die jeweiligen Anpassungen im Verzeichnis »contao/dca« erwartet. Falls die Verzeichnisse noch 
+nicht existieren, musst du diese zunächst erstellen. Für jede Contao Tabelle benötigst du eine eigene Datei, 
+beispielsweise »contao/dca/tl_content.php«. Anschließend mußt du den Anwendungs-Cache über den 
+[Contao-Manager](/de/installation/contao-manager/) oder über die Konsole neu aufbauen. Dieser Schritt ist nach
```
Comment (netzarbeiter) [inline docs/manual/guides/dca-adjustments.de.md]:
```suggestion
[Contao Manager](/de/installation/contao-manager/) oder über die Konsole neu aufbauen. Dieser Schritt ist nach 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**